### DISCUSSION TITLE
Add a Standard User

### DISCUSSION THREAD

**Discussion by alan-m1** [netbox-community/netbox#6384] [General]
Hi,
Apologies if I'm asking this in the wrong place .

I've just installed Netbox on CentOS and I only seem to have the superuser login available to me.   How do I add users, and is it possible to restrict their access to Netbox to read-only ?

THanks

**Comment by candlerb** [comment]
Select "Admin" from the "Admin" menu to get into the Django administration subsite, which lets you create users and groups.

Normal users by default have no permissions at all.  You have to grant them permissions even to view things.  This is described in the manual [here](https://netbox.readthedocs.io/en/stable/administration/permissions/).  Normally it's best to add permissions to a group, and assign users to one or more groups.

As a short-cut, if you want to grant all users "read permission to everything", you can set

```
EXEMPT_VIEW_PERMISSIONS = ['*']
```

in `configuration.py`.  That is documented [here](https://netbox.readthedocs.io/en/stable/configuration/optional-settings/#exempt_view_permissions).

**Reply by alan-m1** [reply]
Thanks candlerb, Should have spotted that.
